src/python/orb.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+
+executors:
+  default:
+    parameters:
+      tag:
+        default: 3.6.2-stretch-browsers
+    docker:
+      - image: circleci/python:<< parameters.tag >>
+
+commands:
+  install-dependencies:
+    parameters:
+      requirements-file:
+        type: string
+        default: "requirements.txt"
+      cache:
+        type: boolean
+        default: true
+    steps:
+      - when:
+          condition: << parameters.cache >>
+          steps:
+            - restore_cache:
+                key: pydeps-{{ .Branch }}-{{ checksum "<< parameters.requirements-file >>" }}
+      - run:
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -r << parameters.requirements-file >>
```

Comment:
> I believe these days pipenv is the new way to do this

That may still be subject to interpretation.

[As commented by Thea, of the Python Packaging Authority](https://www.reddit.com/r/Python/comments/8jd6aq/why_is_pipenv_the_recommended_packaging_tool_by/dyzduci/):

> Pipenv is recommended specifically for dealing with application dependencies. It's not recommend (or even designed for) dealing with library dependencies or to be some magical panacea to solve the universe of Python packaging. 

Though, based on my experience, `pipenv` _can_ be used for libraries, it's just not a perfect experience since you end up with dependencies listed twice, once in your `setup.py` file, and then again in your `Pipfile`.

Could multiple package managers be supported, similar to the `node` Orb?